src/Begin.f90

```diff
@@ -137,8 +183,27 @@ Subroutine BEGIN(param_file,spatial_file)
 !     Variable ndelta read in here.  At present, number of elements
 !     is entered manually into the network file (UW_JRY_2011/03/15)
 !
-    read(90,'(5x,i5,5x,i5,8x,i5,6x,a8,6x,a10,7x,f10.0,f5.0)')  &
+
+        if(reservoir) then
+           read(90,'(5x,i5,5x,i5,8x,i5,6x,a8,6x,a10,7x,f10.0,f5.0,i6)')  &  !nodes for each reach
+              node,nrow,ncol,lat,long,rmile1,ndelta(ncell),node_res
+           if(node_res .gt.0) then
+             res_presx = .true.
+           else
+             res_presx = .false.
+           end if
+
+        !      X,Y matrix with nodes in each reach
+            nodes_x(nr,ncell) = node    ! matrix of nodes for each cell
+            res_num(nr,ncell) = node_res   ! matrix of reservoir index values
+            res_pres(nr,ncell) = res_presx  ! matrix for if reservoir is present for each cell
+            rmile_node(ncell) = rmile1
+
```

Comment:
Style problem again.
Within an `if` loop, you use 4 blanks in the `if` loop between L101 and L115 but you use 2 or 3 blanks in the `if` loop above. Meanwhile, there is one more blank in front of line 197 compared to line 194. I think they should match. You'd better read through the code and fix problems like these.